src/api/index.ts

```diff
@@ -53,6 +58,20 @@ const getParameters = (parameters?: a.Parameter[]) => {
   });
 };
 
+const getRequestBody = (
```

Comment:
I'm not sure I'm completely into the terminology, but isn't this `getRequestBodyResultNode` or something?

Reply:
I seem to have named all these get functions to reflect what is retrieved from the api specification tree. 

E.g. in this case its named after https://swagger.io/docs/specification/describing-request-body/

We could change this terminology since I agree that it is a bit confusing. 

I renamed another function that had the name node just to make the naming at least consistent